README.md

```diff
@@ -1,70 +1,42 @@
-# XFBench: A Cross-Cloud Benchmark Suite for Evaluating FaaS Workflow Platforms
```

Comment:
pls merge lates CCGRID2024 in this branch, and accept README from the latest version.